setup.cfg

```diff
@@ -63,6 +63,9 @@ filterwarnings =
     error:The SparseArray:FutureWarning
 junit_family=xunit2
 
+[codespell]
+ignore-words-list=ba,blocs,coo,datas,fo,hist,nd,ser
```

Comment:
these here, let's open an issue and try to fix some of these 
maybe 'fo' rest might be ok